src/js/control-bar/progress-control/progress-control.js

```diff
@@ -4,6 +4,8 @@
 import Component from '../../component.js';
```

Comment:
We need to do an update in `progressControl` rather then `seekBar` because `progressControl` is the component that is tabbed to and moused over.